2021-09-20 22:17:36,512 INFO    MainThread:536453 [wandb_setup.py:_flush():69] setting env: {}
2021-09-20 22:17:36,512 INFO    MainThread:536453 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-20 22:17:36,512 INFO    MainThread:536453 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-20 22:17:36,512 INFO    MainThread:536453 [wandb_init.py:_log_setup():342] Logging user logs to /tf/CI/MyTorchTemplate/wandb/run-20210920_221736-37gmjjuv/logs/debug.log
2021-09-20 22:17:36,512 INFO    MainThread:536453 [wandb_init.py:_log_setup():343] Logging internal logs to /tf/CI/MyTorchTemplate/wandb/run-20210920_221736-37gmjjuv/logs/debug-internal.log
2021-09-20 22:17:36,512 INFO    MainThread:536453 [wandb_init.py:init():375] calling init triggers
2021-09-20 22:17:36,512 INFO    MainThread:536453 [wandb_init.py:init():382] wandb.init called with sweep_config: {}
config: {'architecture': 'single-layer-perceptron', 'dataset': 'MNIST-handwrite'}
2021-09-20 22:17:36,512 INFO    MainThread:536453 [wandb_init.py:init():424] starting backend
2021-09-20 22:17:36,512 INFO    MainThread:536453 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-09-20 22:17:36,519 INFO    MainThread:536453 [backend.py:ensure_launched():135] starting backend process...
2021-09-20 22:17:36,523 INFO    MainThread:536453 [backend.py:ensure_launched():140] started backend process with pid: 536499
2021-09-20 22:17:36,523 INFO    MainThread:536453 [wandb_init.py:init():429] backend started and connected
2021-09-20 22:17:36,524 INFO    MainThread:536453 [wandb_init.py:init():477] updated telemetry
2021-09-20 22:17:36,524 INFO    MainThread:536453 [wandb_init.py:init():500] communicating current version
2021-09-20 22:17:37,022 INFO    MainThread:536453 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.2 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-09-20 22:17:37,022 INFO    MainThread:536453 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-09-20 22:17:37,405 INFO    MainThread:536453 [wandb_init.py:init():540] starting run threads in backend
2021-09-20 22:17:38,286 INFO    MainThread:536453 [wandb_run.py:_console_start():1601] atexit reg
2021-09-20 22:17:38,287 INFO    MainThread:536453 [wandb_run.py:_redirect():1475] redirect: SettingsConsole.REDIRECT
2021-09-20 22:17:38,287 INFO    MainThread:536453 [wandb_run.py:_redirect():1480] Redirecting console.
2021-09-20 22:17:38,288 INFO    MainThread:536453 [wandb_run.py:_redirect():1536] Redirects installed.
2021-09-20 22:17:38,288 INFO    MainThread:536453 [wandb_init.py:init():565] run started, returning control to user process
2021-09-20 22:17:40,761 INFO    MainThread:536453 [wandb_run.py:_atexit_cleanup():1571] got exitcode: 1
2021-09-20 22:17:40,761 INFO    MainThread:536453 [wandb_run.py:_restore():1543] restore
2021-09-20 22:17:43,307 INFO    MainThread:536453 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 735
  total_bytes: 735
}

2021-09-20 22:17:43,703 INFO    MainThread:536453 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 735
  total_bytes: 735
}

2021-09-20 22:17:44,417 INFO    MainThread:536453 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 735
  total_bytes: 6023
}

2021-09-20 22:17:44,518 INFO    MainThread:536453 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 735
  total_bytes: 6023
}

2021-09-20 22:17:44,619 INFO    MainThread:536453 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 735
  total_bytes: 6023
}

2021-09-20 22:17:44,720 INFO    MainThread:536453 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 735
  total_bytes: 6023
}

2021-09-20 22:17:44,823 INFO    MainThread:536453 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6023
  total_bytes: 6023
}

2021-09-20 22:17:44,927 INFO    MainThread:536453 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6023
  total_bytes: 6023
}

2021-09-20 22:17:45,030 INFO    MainThread:536453 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6023
  total_bytes: 6023
}

2021-09-20 22:17:45,133 INFO    MainThread:536453 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6023
  total_bytes: 6023
}

2021-09-20 22:17:45,236 INFO    MainThread:536453 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6023
  total_bytes: 6023
}

2021-09-20 22:17:45,463 INFO    MainThread:536453 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6023
  total_bytes: 6023
}

2021-09-20 22:17:45,566 INFO    MainThread:536453 [wandb_run.py:_wait_for_finish():1693] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6023
  total_bytes: 6023
}

2021-09-20 22:17:46,734 INFO    MainThread:536453 [wandb_run.py:_show_files():1915] logging synced files
